### DISCUSSION TITLE
ScriptReference/MonoBehaviour.OnRenderImage

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#16] [Notes]
# ScriptReference/MonoBehaviour.OnRenderImage

https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnRenderImage.html

**Comment by vertxxyz** [comment]
This is not called in Edit Mode without the [`ExecuteInEditMode`](https://docs.unity3d.com/ScriptReference/ExecuteInEditMode.html) attribute.
